graylog.js

```diff
@@ -251,7 +251,7 @@ graylog.prototype.send = function (chunk, server, cb) {
 
     this._unsentChunks += 1;
 
-    client.send(chunk, 0, chunk.length, server.port, server.host, function (err/*, bytes */) {
```

Comment:
it's functionally identical... the only reason for getting rid of it is that it's shorter, and some code highlighters trip on it.